src/status_im2/navigation/util.cljs

```diff
@@ -0,0 +1,42 @@
+(ns status-im2.navigation.util
```

Comment:
I didn't know where to add this new function without creating existing or new dependency problems, so I created this namespace.

Please take a look at the function and give any feedback you think will enhance it.

Do you think a macro would be good?
we could avoid repetition and I like it, but at the same time, I don't like existing macros like `defview` that we had in the previous codebase since they add a layer of complexity to our components.
wdyt? :thinking: